README.Rmd

```diff
@@ -245,10 +253,77 @@ ADF_dtype |>
 
 ```
 
+## Spatial Analysis
+
+```{r}
+library(sf)
+library(tmap)
+```
+The following code reads the network with the estimated commute trips from the npt repository
+
+```{r}
+rnet_commute = read_rds("../npt/outputs/rnet_commute.Rds")
+rnet_commute
+```
+A `sf` object is created from the `clean_counts` data frame.
+
+```{r}
+sf_counts = clean_counts |>
+  select(siteID,latitude,longitude,provider,location) |>
+  unique() |>
+  st_as_sf(coords = c("longitude","latitude"),crs = 4326)
+sf_counts
+```
+
+
+
+A subset of the counts are taken based on a buffer of the `rnet_commute` object.
+
+```{r}
+rnet_buffer20 = rnet_commute |> st_buffer(dist = 20)
```

Comment:
That's reasonable. May be best to union the object first. How long did it take to run?

Reply:
The buffer takes 1.4 minutes to run. The rest of the code does not take more than 10 seconds.

Reply:
I guess it will be quicker if you do...

Reply:
```suggestion
rnet_buffer20 = rnet_commute |> st_union() |> st_buffer(dist = 20)
```

1+ minute is a lot.